src/ploneorg/addonlisting/browser/views.py

```diff
@@ -50,19 +48,19 @@ def all_addons(self):
         )
 
     def downloaded_addons(self):
-        context = aq_inner(self.context)
-        catalog = api.portal.get_tool(name='portal_catalog')
-        return catalog(
-            object_provides=IAddOn.__identifier__,
-            path='/'.join(context.getPhysicalPath()),
+        return api.content.find(
+            context=self.context,
+            depth=1,
+            portal_type='AddOn',
             sort_on='download_sum_total',
```

Comment:
that will propably raise an error as the catalog did not scan the fields but the indexed catalog equivalents 

so for example title become sortable_title
those fields are not in the index